docs/source/api/apollo-server.mdx

```diff
@@ -723,168 +569,4 @@ If `query` contains more than one operation definition, you must provide this op
 </tr>
 
 </tbody>
```

Comment:
Missing docs for:
- `startInBackgroundHandlingStartupErrorsByLoggingAndFailingAllRequests`
- `executeHTTPGraphQLRequest`
- The `cache` and `logger` readonly fields

Reply:
Added sections for all four of these!